packages/chart/src/components/EditorPanel/components/Panels/Panel.Visual.tsx

```diff
@@ -164,6 +164,34 @@ const PanelVisual: FC<PanelProps> = props => {
           config.visualizationType === 'Combo') ||
           config.visualizationType === 'Line') && (
           <>
+            <Select
+              tooltip={
+                <Tooltip style={{ textTransform: 'none' }}>
+                  <Tooltip.Target>
+                    <Icon
+                      display='question'
+                      style={{ marginLeft: '0.5rem', display: 'inline-block', whiteSpace: 'nowrap' }}
+                    />
+                  </Tooltip.Target>
+                  <Tooltip.Content>
+                    <p>
+                      Shapes will display in order circle, square, triangle, diamond, inverted triangle. Use with no
```

Comment:
This might read better:

Shapes will appear in the following order: circle, square, triangle, diamond, and inverted triangle. Use with a maximum of 5 data points.